wsd/COOLWSD-main.cpp

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Not clear to me why we need all these new files. Is there an explanation on the design somewhere?

Reply:
@Ashod comment above will explain: https://github.com/CollaboraOnline/online/pull/7329#pullrequestreview-1655289137